apps/omg_watcher_info/test/omg_watcher_info/pending_block_processor_test.exs

```diff
@@ -0,0 +1,122 @@
+# Copyright 2019-2020 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.WatcherInfo.PendingBlockProcessorTest do
+  use ExUnitFixtures
+  use ExUnit.Case, async: false
+
+  import OMG.WatcherInfo.Factory
+
+  alias OMG.WatcherInfo.DB
+  alias OMG.WatcherInfo.DB.PendingBlock
+  alias OMG.WatcherInfo.PendingBlockProcessor
+
+  @interval 500
+
+  setup_all tags do
+    {:ok, pid} =
+      GenServer.start_link(
+        PendingBlockProcessor,
```

Comment:
`PendingBlockProcessor` is has a genserver behaviour, as suck, it already exposes start_link/1. Use that. It's more common to use the exposed API.